README.md

```diff
@@ -10,3 +10,6 @@
 - `bundle exec rake db:seed` - Seed with starter data
 - `bundle exec rails s` - Serve the application
 - `bundle exec rake db:reset` - If you want to reset and re-seed your db
+
+## Additonal Docs
+* [Frontend developer workflow](https://github.com/coderie/coderie-site/blob/master/docs/fontend-workflow.md)
```

Comment:
Can you change this to be a URI rather than URL so we can point to whatever branch and optionally change URL structure (also work on PR's)?  I think `/docs/frontend-workflow.md` will do it.

Also noticed there is a typo in the URL, it read "`fontend-workflow`"